### PR TITLE
Changes post-sale artwork supplementary info

### PR DESCRIPTION
**Pull request by ashfurrow**
This is work for https://github.com/artsy/auctions/issues/531 and the Emission equivalent of Eigen's https://github.com/artsy/eigen/pull/2402

Not sure about the yarn.lock file changes – please take a look.

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  No CHANGELOG added.
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: src/lib/Components/ArtworkGrids/Artwork.tsx [hunk 1]
```diff
@@ -63,20 +63,24 @@ class Artwork extends React.Component<RelayProps, any> {
...
-      const numberOfBids = artwork.sale_artwork.bidder_positions_count
-      let text = artwork.sale_artwork.opening_bid.display
-      if (numberOfBids > 0) {
-        text = `${artwork.sale_artwork.current_bid.display} (${numberOfBids} bid${numberOfBids === 1 ? "" : "s"})`
+    if (artwork.is_in_auction) {
+      if (artwork.sale_artwork.sale.is_open) {
+        const numberOfBids = artwork.sale_artwork.bidder_positions_count
+        let text = artwork.sale_artwork.opening_bid.display
+        if (numberOfBids > 0) {
+          text = `${artwork.sale_artwork.current_bid.display} (${numberOfBids} bid${numberOfBids === 1 ? "" : "s"})`
```
Comment (orta) [inline src/lib/Components/ArtworkGrids/Artwork.tsx]: could this go in metaphysics as a display string?

Comment (ashfurrow) [inline src/lib/Components/ArtworkGrids/Artwork.tsx]: It _could_ but eigen would still need to compute it manually (the part in parens is in a different font colour).

**Comment by orta** [comment]
Yeah, this looks 👍

**Pull request merged**